teams/team_2/training/train.py

```diff
@@ -57,12 +83,11 @@ def forward(self, features):
         return score
 
 
-def evaluate_fitness(task_model: nn.Module, rest_model: nn.Module):
-    # for task in task_environment:
-    # action = model(task.features, task.state)
-    result = run(task_model, rest_model)
-    # result = task_environment.run(action)  # Simulates environment behavior
-    return result
+def evaluate_fitness(task_model: nn.Module, rest_model: nn.Module, turns, civilians):
+    tasks = run(task_model, rest_model, turns, civilians)
+    score = tasks / turns / civilians
+    print(f"score: {round(score, 3)}")
+    return score
```

Comment:
redefine score as `tasks / turns / civilians (members)`, denotes how many tasks were completed per member in a single turn on average